.eslintrc

```diff
@@ -43,7 +43,7 @@
   },
   "overrides": [
     {
-      "files": ["*.stories.js"],
+      "files": ["*.stories.js", "**/stories/**"],
```

Comment:
NICE! we might want to move the Chart stories in the same way.

Reply:
At some point, yes.